src/pymor/algorithms/newton.py

```diff
@@ -124,6 +128,15 @@ def newton(operator, rhs, initial_guess=None, mu=None, error_norm=None, least_sq
             correction = jacobian.apply_inverse(residual, least_squares=least_squares)
         except InversionError:
             raise NewtonError('Could not invert jacobian')
+        if line_search_method is not None:
+            if line_search_method == 'armijo':
```

Comment:
Add this point I would add a logger.info such that the user knows, that a line_search method has been used.